File: virtual-trading-card-game/Admin/CardCreatorCustom.java
package Admin;

import java.util.Scanner;

public class CardCreatorCustom extends CardCreator{

	private Scanner scanner = new Scanner(System.in);
	String input;
	int intInput;
	CardCustom cardCustom = new CardCustom();

	CardCreatorCustom(){

		System.out.println("Please Enter the Card Name:");
		input = scanner.nextLine();
		cardCustom.setName(input);
		System.out.println("Please Enter the Card Red Stat:");
		intInput = scanner.nextInt();
		cardCustom.setRedStat(intInput);
		System.out.println("Please Enter the Card Blue Stat:");
		intInput = scanner.nextInt();
		cardCustom.setBlueStat(intInput);
		System.out.println("Please Enter the Card Green Stat:");
		intInput = scanner.nextInt();
		cardCustom.setGreenStat(intInput);
		System.out.println("Please Enter the Rarity:");
		input = scanner.nextLine();

		input = scanner.nextLine();
		cardCustom.setRarity(input);
		System.out.println("Please Enter the Card Weakness:");
		input = scanner.nextLine();
		cardCustom.setWeakness(input);
		System.out.println("Please Enter the Card Strength:");
		input = scanner.nextLine();
		cardCustom.setStrength(input);
		System.out.println("Please Enter the Card Type:");
		input = scanner.nextLine();
		cardCustom.setType(input);
		cardCustom.displayProps();
	}
}

File: virtual-trading-card-game/PythonAdmin/CardCreatorCustom.py
from CardCreator import CardCreator 
from CardCustom import CardCustom

class CardCreatorCustom(CardCreator):

	Card = CardCustom()

	def __init__(self):
		userInput = input("Please Enter the Card Name: ")
		self.Card.setName(userInput)
		userInput = input("Please Enter the Card Red Stat: ")
		self.Card.setRedStat(userInput)
		userInput = input("Please Enter the Card Blue Stat: ")
		self.Card.setBlueStat(userInput)
		userInput = input("Please Enter the Card Green Stat: ")
		self.Card.setGreenStat(userInput)
		userInput = input("Please Enter the Card Rarity: ")
		self.Card.setRarity(userInput)
		userInput = input("Please Enter the Card Weakness: ")
		self.Card.setWeakness(userInput)
		userInput = input("Please Enter the Card Strength: ")
		self.Card.setStrength(userInput)
		userInput = input("Please Enter the Card Type: ")
		self.Card.setType(userInput)
		self.Card.displayProps()
		self.Card.upload_to_database()
	

